File: 백준/Python/Bronze/5622. 다이얼/다이얼.py
number = ['0', '0', 'ABC', 'DEF', 'GHI', 'JKL', 'MNO', 'PQRS', 'TUV', 'WXYZ']

s = input()

total = 0
for w in s:
    for n in number:
        if w in n:
            total += (number.index(n)+1)
            break

print(total)

File: 백준/Java/Bronze/5622. 다이얼/다이얼.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        
        String s = sc.next();
        String[] number = {"", "", "ABC", "DEF", "GHI", "JKL", "MNO", "PQRS", "TUV", "WXYZ"};
        
        int total = 0;
        for(int i=0; i<s.length(); i++) {
            for(int j=0; j<10; j++) {
                for(int k=0; k<number[j].length(); k++) {
                    if(s.charAt(i) == number[j].charAt(k)) {
                        total += j+1;
                        break;
                    }
                }
            }
        }
        
        System.out.print(total);
    }
}